README.md

```diff
@@ -102,6 +102,10 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [What is Appwrite and how to contribute for Hacktoberfest](https://dev.to/mishmanners/four-projects-to-contribute-to-this-hacktoberfest-pep)
 * [Appwrite, giving it a shot this Hacktoberfest ‘21](https://medium.com/@anand26shweta/appwrite-giving-it-a-shot-ad1ef79ae44f)
 
+## Hacktoberfest 2022
+* [Appwrite, Hacktoberfest, and how to contribute, including AMA with viewers](https://www.twitch.tv/videos/1634653922)
+* [Featured on the GitHub Blog](https://github.blog/release-radar-sept-2022)
```

Comment:
I get a

> Whoops, we haven't written that blog post yet!

error when trying to browse to this.

Reply:
it should be there now. It was caching through from the WP side. Can you check it now 😄

Reply:
@stnyugen90 can you please check now.